api-compact.yaml

```diff
@@ -66,6 +59,41 @@ apigateway_responses_with_mapping_template: &apigateway_responses_with_mapping_t
   500:
     statusCode: "500"
 
+apigateway_catalog_courses_response_with_mapping_template: &apigateway_catalog_courses_response_with_mapping_template
```

Comment:
Do we need two separate templates? Couldn't we run the same string replace on all endpoints?